src/components/TodayRoutine/TodayRoutine.js

```diff
@@ -0,0 +1,174 @@
+import styled from 'styled-components';
+import RoutineThumbNail from '../RoutineThumbNail/RoutineThumbNail';
+import { useEffect } from 'react';
+import { useState } from 'react';
```

Comment:
동일 패키지의 import 내용은 한 줄로 써주시면 됩니다.
그리고 import 순서 컨벤션 지켜주세요!
```suggestion
import { useEffect, useState } from 'react';
import styled from 'styled-components';
import RoutineThumbNail from '../RoutineThumbNail/RoutineThumbNail';
```